new_platforms/vscode-extension/package.json

```diff
@@ -90,6 +90,7 @@
 		"vscode-extension-telemetry": "0.0.18"
 	},
 	"extensionDependencies": [
+		"ms-vscode.cpptools",
```

Comment:
Could we also add a dependency to https://marketplace.visualstudio.com/items?itemName=MicrosoftDebuggingPlatform.vscode-cdb in this PR or another?

Reply:
Ah, I forgot, will do.

Reply:
Added.